src/styles/login.scss

```diff
@@ -7,6 +7,8 @@ $white: rgb(255,255,255);
     padding: 3%;
     background-color: $primary;
     color: $white;
+    display: flex;
+    flex-direction: column;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/nezago/phone-history-store-frontend/pullRequest?prid=6016707)